### PR TITLE
Update types and components

### PR DESCRIPTION
**Pull request by bmigirl**
I want to merge this change because it adds handling of new description and content fields instead of old json fields

[ticket](https://app.clickup.com/t/2549495/SALEOR-2170)

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2170 Drop deprecated JSON fields in Storefront/SDK](https://app.clickup.com/t/2549495/SALEOR-2170)

**Review by krzysztofwolski — APPROVED** [review]
The description seems to be displaying right 👍

**Comment by krzysztofwolski** [comment]
(but jest tests are falling!)

**Comment by piotrgrundas** [comment]
U can bump editorjs and enable readOnly mode. They resolved the relevant issue 👌

File: src/@next/components/molecules/ProductDescription/ProductDescription.tsx [hunk 1]
```diff
@@ -49,8 +48,8 @@ export const ProductDescription: React.FC<IProps> = ({
         </S.TabTitle>
       </S.Tabs>
       <div hidden={activeTab !== TABS.DESCRIPTION}>
-        {descriptionJson ? (
-          <RichTextEditorContent jsonData={descriptionJson} />
+        {description ? (
+          <RichTextEditorContent jsonData={description} />
         ) : (
           <p>{description}</p>
         )}
```
Comment (piotrgrundas) [inline src/@next/components/molecules/ProductDescription/ProductDescription.tsx:55]:
```suggestion
        {description && (
          <RichTextEditorContent jsonData={description} />
        )}
```
Current check doesn't make sense 🤷

**Comment by mmarkusik** [comment]
Along with @krzysztofwolski we weren't able to figure out why enzyme doesn't render the editor content correctly (instead it renders jsx tags and no jsx / text content) so updated the test to simple snapshot since that's all it did anyway.

File: src/@next/components/atoms/Button/styles.ts [hunk 2]
```diff
@@ -55,6 +55,7 @@ export const Primary = styled.button<{
   `}
 `;
 
+// @ts-ignore
```
Comment (dominik-zeglen) [inline src/@next/components/atoms/Button/styles.ts:58]: Why?

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**